test/azure/AcceptanceTests/zzz_tests.py

```diff
@@ -65,7 +66,7 @@ def test_ensure_coverage(self):
 
         totalTests = len(report)
         print("The test coverage is {0}/{1}.".format(totalTests - len(skipped), totalTests))
-        self.assertEqual(0, len(skipped))
+        # self.assertEqual(0, len(skipped))
```

Comment:
Not the case (at least currently). But also, the future plan is to reference `testserver@^2.0.0` to *automatically* pick up and test against recent features we are expected to have! So maybe, commenting this out is actually the right thing to do long term - your coverage will always be clearly visible on PRs anyways.